samples/java/hibernate/src/main/java/com/google/cloud/postgres/models/Venues.java

```diff
@@ -39,8 +40,9 @@ public class Venues {
   @Column(name = "name", nullable = false)
   private String name;
 
-  @Column(name = "description", nullable = false)
-  private String description;
+  @Column(name = "description", nullable = false, columnDefinition = "jsonb")
```

Comment:
Has the script that creates the sample data model been updated? Or is that not needed?

Reply:
Done.